### PR TITLE
OHRI-1237 Mother MNCH Summary revised

### PR DESCRIPTION
**Pull request by kajambiya**

### CODE REVIEW

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx [hunk 1]
```diff
@@ -149,14 +172,34 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
     return items;
   }, [relatives, relativeToIdentifierMap]);
 
+  const childrenDetails: pregnancyOutcomeProps[] = useMemo(() => {
+    let items = [];
+    pregnancyOutcomes.forEach((child) => {
+      let infantStatusObs = child.groupMembers.find((member) => member.concept.uuid === infantStatusAtBirthConcept);
+      let childObject: pregnancyOutcomeProps = {
+        id: child.groupMembers.find((member) => member.concept.uuid === infantPTrackerIdConcept).value,
```
Comment (pirupius) [inline packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx:180]: This line looks like the data being displayed is actually the `pTracker` and not the id (uuid). My view is that u change this to pTrackerId for consistency

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx [hunk 2]
```diff
@@ -83,7 +88,7 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const headersPregnancyOutcome = [
     {
       header: t('pTrackerId', 'PTracker ID'),
-      key: 'pTrackerId',
+      key: 'id',
```
Comment (pirupius) [inline packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx:91]: Same comment applies here, let's maintain `pTrackerId`

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx [hunk 3]
```diff
@@ -161,7 +163,8 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
         </Link>
       );
       let relativeObject: familyItemProps = {
-        id: relativeToIdentifierMap.find((entry) => entry.patientId === relative.personB.uuid)?.pTrackerId,
+        id: i,
```
Comment (pirupius) [inline packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx:166]: i thought we was using the uuid? there's a danger with using interating numbers with such lists

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx [hunk 4]
```diff
@@ -174,12 +177,13 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
 
   const childrenDetails: pregnancyOutcomeProps[] = useMemo(() => {
     let items = [];
-    pregnancyOutcomes.forEach((child) => {
+    pregnancyOutcomes.forEach((child, i) => {
       let infantStatusObs = child.groupMembers.find((member) => member.concept.uuid === infantStatusAtBirthConcept);
       let childObject: pregnancyOutcomeProps = {
-        id: child.groupMembers.find((member) => member.concept.uuid === infantPTrackerIdConcept).value,
+        id: i,
```
Comment (pirupius) [inline packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx:183]: same applies here

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/hiv-exposed-infant.component.tsx [hunk 5]
```diff
@@ -180,7 +179,8 @@ const HivExposedInfant: React.FC<{
         </Link>
       );
       let relativeObject: familyItemProps = {
-        id: relativeToIdentifierMap.find((entry) => entry.patientId === relative.personA.uuid)?.pTrackerId,
+        id: i,
```
Comment (pirupius) [inline packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/hiv-exposed-infant.component.tsx:182]: same comment

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx (hunk 4 above)
Comment (kajambiya) [reply packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx]: done

**Review by pirupius — APPROVED** [review]

**Pull request merged**